invenio_files_rest/serializer.py

```diff
@@ -243,6 +243,16 @@ def wait_for_taskresult(task_result, content, interval, max_rounds):
 
     The task will periodically send whitespace to prevent the connection from
     being closed.
+
+    :param task_result: The async task to wait.
+    :param content: The content to return when the task is ready.
+    :param interval: The period of time to sleep before check again if the task
+        is ready.
+    :param max_rounds: The maximum number of time the function check before
+        raise an Exception.
+    :returns: An iterator for the content or a
+        :class:`invenio_files_rest.errors.FilesException` exception if the
+        timeout happened or the job fails.
```

Comment:
247: to wait for.
249: The duration of a sleep period before checking again...
251: The maximum duration of the check before raising an Exception.
253: An iterator on the content... or the job failed.